doc/adding_new_libraries.md

```diff
@@ -9,54 +9,88 @@ which libraries are based on gRPC.
 We will use `tasks` as an example through this document, you will need to
```

Comment:
It looks like `tasks` is now only referenced in one place, and perhaps that should be changed to `${library}`.

Reply:
Fixed, thanks.